### PR TITLE
Fix k8s ebs bounce

### PR DESCRIPTION
**Pull request by mattmb**
* pin k8s service instances with EBS to downthenup bounce method
* pin k8s service instances with EBS to instances=1

### CODE REVIEW

**Comment by vkhromov** [comment]
TBH I'm not sure that this method is OK, since we'll implicitly override users' configs setting, so they probably would be surprised by i.e. discovering a single running instance after specifying something like `instances: 3` without any notifications.
Probably it's better to have pre-commit hook and to explicitly explain to users that they need to fix in their configs.

**Comment by Rob-Johnson** [comment]
I think I agree - we should refuse to deploy (and hopefully catch the problem in pre-commit) if we had unmatchable constraints, rather than changing the settings without warning.

File: paasta_tools/kubernetes_tools.py [hunk 1]
```diff
@@ -246,6 +246,10 @@ def copy(self) -> "KubernetesDeploymentConfig":
     def get_bounce_method(self) -> str:
         """Get the bounce method specified in the service's kubernetes configuration."""
         # map existing bounce methods to k8s equivalents.
+        # but if there's an EBS volume we must downthenup to free up the volume.
+        # in the future we may support stateful sets to dynamically create the volumes
+        if self.get_aws_ebs_volumes():
+            return 'Recreate'
```
Comment (keymone) [inline paasta_tools/kubernetes_tools.py]: KUBE_DEPLOY_STATEGY_MAP['downthenup']

just to be consistent with the line below

or even:

    if self.get_aws_ebs_volumes():
        strategy = 'downthenup'
    else:
        strategy = self.config_dict.get('bounce_method', 'crossover')
    return KUBE_DEPLOY_STATEGY_MAP[strategy]

**Comment by mattmb** [comment]
Ok, makes sense to me, we should raise an error or something. Maybe I'll revisit the error handling for `format_kubernetes_deployment` in general.

**Pull request merged**